### DISCUSSION TITLE
On raiding ID or raiding galore

### DISCUSSION THREAD

**Discussion by razielfarid2** [chromiecraft/chromiecraft#7372] [Suggestions]
Hey. Sorry in advance if this has already been suggested. I did not find an exactly similar suggestion.

How about you guys do a favor to (kinda) autistic people like me and give us a raid ID system just like that of the worldbosses'. Nothing too fancy, just not being able to access loots if one is under an ID for the week. I thought it would help with raid filling with nowadays' population. 

I would love to abuse raiding for the simple reason that I love the state of my class/spec at the moment and it'd be good training for one to get used to a raid's mechanics. 

Thank you for reading me.

**Comment by Belodina** [comment]
Hey there! This is an interesting idea, but not likely to be implemented I feel. If the change were to happen, most people wouldn't engage with it since they won't be able to get loot, or only have time for 1 or 2 raid nights a week and will want to make the most of them across 1 or more characters. While a World Boss is very quick to hop into a group and help your guild out, even if you are locked, a raid requires a bit more effort and time investment. Still a very interesting suggestion thank you!! Hopefully we have seen the worst of the low population, and people will start returning towards the end of TBC :3

**Comment by krutoj87** [comment]
This system would have pros, but it would also have cons. Let's say your guild cleared the raid and everybody got their items. Now you just can sell clear runs to other players, they will get 100% of the loot and your guild already has their items for this ID.

Also it could make it harder for some people to get raid spots in PUGs, because they might be taken by people running the raid for the second or third time this week. 

And what would happen if you have an ID with 4/10 bosses down and join a full clear run. It would not only cause headaches for the devs to figure out how to handle it, it would also cause headaches for PUGs how to handle loot rules in those situations.

**Comment by razielfarid2** [comment]
:(
